pom.xml

```diff
@@ -102,13 +101,15 @@
                     </additionalOptions>
                 </configuration>
             </plugin>
+            <!-- Do not deploy anything except for the plugin -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
                     <compilerArgs>
                         <arg>-Xlint:all</arg>
                     </compilerArgs>
+                    <skip>true</skip>
```

Comment:
This might be the same case. Not sure if compiler supports skipping, but we certainly do not want to do that.

Reply:
If I recall correctly, this was for the purpose that we wanted to get a report of all complier/linter issues (therefore 'Xlint:all') but there were also false issues which didn't have a proper solution/work-around. So for that purpose we couldn't stop the build because we have had only two options there:
1. Do all Lint reports but not fast-fail building process (because we couldn't fix all linted reports due to dependencies/transitive dependencies where things are out of our control)
2. Do a particular (probably none) Lint report and have fast-fail for that (if reported issues are really valid)